src/literature/components/AuthorList.jsx

```diff
@@ -1,22 +1,73 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { List } from 'immutable';
+import { Modal } from 'antd';
 
 import InlineList from '../../common/components/InlineList';
 import AuthorLink from './AuthorLink';
 
 class AuthorList extends Component {
+  constructor(props) {
+    super(props);
+    this.onModalCancel = this.onModalCancel.bind(this);
+    this.onModalOpen = this.onModalOpen.bind(this);
+    this.suffix = this.suffix.bind(this);
+
+    this.state = {
+      modalVisible: false,
+    };
+  }
+
+  onModalOpen() {
+    this.setState({ modalVisible: true });
+  }
+
+  onModalCancel() {
+    this.setState({ modalVisible: false });
+  }
+
+  suffix() {
+    const { authors, limit } = this.props;
+    if (authors.size > limit) {
+      return (
+        <div className="di">
+          {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions, jsx-a11y/anchor-is-valid */}
+          <a onClick={this.onModalOpen}> Show all</a>
+        </div>
+      );
+    }
+    return null;
+  }
+
   render() {
+    const { modalVisible } = this.state;
     const { authors, limit, recordId } = this.props;
     return (
-      <InlineList
-        items={authors.take(limit)}
-        suffix={authors.size > limit ? <span> et al.</span> : null}
-        extractKey={author => author.get('full_name')}
-        renderItem={author => (
-          <AuthorLink author={author} recordId={recordId} />
-        )}
-      />
+      <div>
+        <InlineList
+          items={authors.take(limit)}
+          suffix={this.suffix()}
+          extractKey={author => author.get('full_name')}
+          renderItem={author => (
+            <AuthorLink author={author} recordId={recordId} />
+          )}
+        />
+        <Modal
+          title={`${authors.size} authors`}
+          width="50%"
+          visible={modalVisible}
+          footer={null}
+          onCancel={this.onModalCancel}
+        >
+          <InlineList
```

Comment:
this `InlineList` too similar with the above one, perhaps we can move this logic to another method.